Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/source/InMemoryRemoteTaskDataSource.java

```diff
@@ -150,7 +150,25 @@ public void call(Task task) {
                 dataSourceTasks.remove(taskId);
                 return Observable.empty();
             }
-        }).delay(remoteDelay, remoteDelayUnit, Schedulers.immediate());
+        }).compose(this.<Void>delay());
+    }
+
+    private <T> Observable.Transformer<T, T> delay() {
```

Comment:
This kind of looks like a duplicate of `InMemoryLocalTaskDataSource.delay()`, maybe it can be extracted to a class?